doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -569,10 +599,9 @@ Induction
 
    Beware that injection yields an equality in a sigma type whenever the
    injected object has a dependent type :g:`P` with its two instances in
-   different types :g:`(P t`:sub:`1` :g:`... t`:sub:`n` :g:`)` and
-   :g:`(P u`:sub:`1` :g:`... u`:sub:`n` :sub:`)`. If :g:`t`:sub:`1` and
-   :g:`u`:sub:`1` are the same and have for type an inductive type for which a decidable
-   equality has been declared using :cmd:`Scheme` :n:`Equality ...`
+   different types :n:`(P t__1 … t__n)` and :n:`(P u__1 … u__n)`. If :n:`t__1` and
+   :n:`u__1` are the same and have for type an inductive type for which a decidable
+   equality has been declared using :cmd:`Scheme` :n:`Equality …`
    (see :ref:`proofschemes-induction-principles`),
    the use of a sigma type is avoided.
```

Comment:
This would deserve to be in a `.. note::`. The ``(see :ref:`proofschemes-induction-principles`)`` is useless and could be removed. It would also make sense to add a reference to the tactic `dependent rewrite` from here, since this is precisely what it is used for.

The note below (which doesn't appear in the diff, so I cannot comment on it) should be gathered with the different types of `induction_arg` exactly like in the case of `discriminate`. But again, I actually doubt that it matches any real use case.